Makefile

```diff
@@ -1,9 +1,12 @@
 all: verify_contracts install
 
-compile_contracts:
+solve_templates:
```

Comment:
I would suggest something like `render_contracts` or `generate_contracts` from the template

Reply:
`render_contracts` sounds good. I'll change the names in this PR.